packages/grid/_modules_/grid/models/gridOptions.tsx

```diff
@@ -312,10 +312,10 @@ export const DEFAULT_GRID_OPTIONS: GridOptions = {
   sortingMode: FeatureModeConstant.client,
   sortingOrder: ['asc', 'desc', null],
   columnTypes: DEFAULT_COLUMN_TYPES,
-  disableColumnMenu: process.env.NODE_ENV === 'production',
-  disableColumnFilter: process.env.NODE_ENV === 'production',
-  disableColumnSelector: process.env.NODE_ENV === 'production',
-  hideToolbar: process.env.NODE_ENV === 'production',
+  // disableColumnMenu: process.env.NODE_ENV === 'production',
+  // disableColumnFilter: process.env.NODE_ENV === 'production',
+  // disableColumnSelector: process.env.NODE_ENV === 'production',
+  // hideToolbar: process.env.NODE_ENV === 'production',
```

Comment:
I guess this is only for testing and before merging these will be uncommented.

Reply:
Yes there is an issue with the CI and the docs so I was checking if that caused it. I will rollback and apply the experimental flag from @oliviertassinari